js/CADWorker/CascadeStudioSceneShapesService.js

```diff
@@ -0,0 +1,10 @@
+export let sceneShapes = [];
+export const resetSceneShapes = () => (sceneShapes = []);
+
+/** This function returns a version of the `inputArray` without the `objectToRemove`. */
+export function RemoveFromSceneShapes(objectToRemove) {
```

Comment:
No longer takes an array as an argument and instead assume you only want to mutate `sceneShapes` I can change it back? Is it likely that users will use this to remove shapes from their own array that aren't sceneShapes?

I also updated the IntelliSense file to suit.